### ISSUE TITLE
Set file type 'asciidoc' for files with .adoc extension

### ISSUE THREAD

**Issue by glts** [vim/vim#417]
`.adoc` is another popular file extension for AsciiDoc files, see for example
http://asciidoctor.org/docs/asciidoc-writers-guide/#content-is-king.

It would be nice to have the file type for `.adoc` files set by default.

``` diff
diff --git a/runtime/filetype.vim b/runtime/filetype.vim
index 923df6e..8c3b203 100644
--- a/runtime/filetype.vim
+++ b/runtime/filetype.vim
@@ -143,7 +143,7 @@ au BufNewFile,BufRead .arch-inventory,=tagging-method   setf arch
 au BufNewFile,BufRead *.art            setf art

 " AsciiDoc
-au BufNewFile,BufRead *.asciidoc       setf asciidoc
+au BufNewFile,BufRead *.asciidoc,*.adoc        setf asciidoc

 " ASN.1
 au BufNewFile,BufRead *.asn,*.asn1     setf asn
```

**Comment by brammool** [comment]
David Burgin wrote:

> `.adoc` is another popular file extension for AsciiDoc files, see for example
> http://asciidoctor.org/docs/asciidoc-writers-guide/#content-is-king.
> 
> It would be nice to have the file type for `.adoc` files set by default.
> 
> ``` diff
> diff --git a/runtime/filetype.vim b/runtime/filetype.vim
> index 923df6e..8c3b203 100644
> --- a/runtime/filetype.vim
> +++ b/runtime/filetype.vim
> @@ -143,7 +143,7 @@ au BufNewFile,BufRead .arch-inventory,=tagging-method setf arch
>  au BufNewFile,BufRead *.art          setf art
> 
>  " AsciiDoc
> -au BufNewFile,BufRead *.asciidoc     setf asciidoc
> +au BufNewFile,BufRead *.asciidoc,*.adoc      setf asciidoc
> 
>  " ASN.1
>  au BufNewFile,BufRead *.asn,*.asn1       setf asn
> ```

Thanks, I'll include it.

## 

FIRST VILLAGER: We have found a witch.  May we burn her?
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\
\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by glts** [comment]
Included with ca63501fbcd1cf9c8aa9ff12c093c95b62a89ed7, thanks.
